default_app/forkedProcess.js

```diff
@@ -0,0 +1,12 @@
+const os = require('os')
+const path = require('path')
+
+let productName = 'Child Product';
+let companyName = 'Child Company';
+let tmpPath = path.join(os.tmpdir(), productName + ' Crashes');
+
+process.startCrashReporter(productName, companyName, 'http://localhost:8080/uploadDump/childDump', tmpPath, {
```

Comment:
@ramya-rao-a I pushed a change to address this where `CrashReporter::StartWithOptions` takes a dictionary and then calls `CrashReporter::Start` with the extracted values as parameters.